tools/timemachine/internal/cmd/downloadHeight.go

```diff
@@ -0,0 +1,226 @@
+// Copyright (c) 2022 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"google.golang.org/api/iterator"
+)
+
+// const value
+var (
+	GcpTimeout  = time.Second * 60
+	ErrNotExist = errors.New("height is not existed.")
+)
+
+// downloadHeight represents the download command
+var downloadHeight = &cobra.Command{
+	Use:   "download",
+	Short: "Download specific height of block",
+	Args:  cobra.ExactArgs(2),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+		height, err := strconv.ParseUint(args[0], 10, 64)
+		if err != nil {
+			return errors.Wrapf(err, "Failed to convert input height: %s.", args[0])
+		}
+		if height == 0 {
+			return errors.New("input height cannot be 0.")
+		}
+		var (
+			heighM    = float64(height) / float64(1024) / float64(1024)
+			heightDir string
+			destDir   = args[1]
+			bucket    = "blockchain-golden"
+			prefix    = "fullsync/mainnet/"
```

Comment:
1. bucket and prefix can be a local constant
2. one var "blockchain-golden/fullsync/mainnet/" should be good enough

Reply:
fixed